testing/ncids-css-testing/stories/components/nci-cta-strip/nci-cta-strip.testcases.long.stories.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { TestCase } from '../../../components/test-case';
+import Component from '@nciocpl/ncids-twig/components/nci-cta-strip/nci-cta-strip.twig';
+import css from './nci-cta-strip.scss';
+
+export default {
```

Comment:
Each story does not need it's own story file. You can combine long and spanish into one.